# This is a service to compute th IK for a given move_it group. 
#
# The group is defined by its name. 
# The desired pose is defined for a specific link. If not set it is supposed to be
# the group end-effector  
# active_joints field specify which joints can be moved. If not set all joints of
# the group are supposed to be active
#
# The service returns the joint state corresponding to the desired pose

geometry_msgs/PoseStamped[] desired_poses
sensor_msgs/JointState fixed_joints
float32 tolerance
string[] group_names
sensor_msgs/JointState seed 
---
sensor_msgs/JointState joint_state 